config/confd/templates/logentries.conf.tmpl

```diff
@@ -1,7 +0,0 @@
-{{if getenv "LOGENTRIES_TOKEN"}}
```

Comment:
Support for Logentries cannot be removed as we're currently using it for production logs. Did you bump into issues with it?

I'll have to double-check that the no-systemd base image actually supports Logentries before switching to it.

Reply:
Right, the base image does not care about Logentries -- all that's needed is configuring rsyslog to send logs there. So, yeah, let's please revert the rsyslog-related changes.